views/partials/featured_breach.hbs

```diff
@@ -4,11 +4,11 @@
         <img alt="{{ featuredBreach.Title }}" src="/img/logos/{{ featuredBreach.Name }}.{{ featuredBreach.LogoType }}">
       </div>
     <div class="breach-info-wrap">
-      <span class="breach-headline medium">{{ featuredBreach.Title }} Breach</span>
-      <span class="breach-info breach-date"><span class="medium">Breach Date</span> {{prettyDate featuredBreach.BreachDate }}</span>
-      <span class="breach-info"><span class="medium">Compromised Accounts</span> {{localeString featuredBreach.PwnCount }}</span>
+      <span class="breach-headline medium">{{ featuredBreach.Title }}</span>
+      <span class="breach-info breach-date"><span class="medium">Breach date:</span>{{prettyDate featuredBreach.BreachDate }}</span>
+      <span class="breach-info"><span class="medium">Compromised accounts:</span>{{localeString featuredBreach.PwnCount }}</span>
       {{#if featuredBreach.DataClasses }}
-      <span class="breach-info"><span class="medium">Compromised Data</span> {{breachDataClasses featuredBreach.DataClasses }}</span>
+      <span class="breach-info"><span class="medium">Compromised data:</span>{{breachDataClasses featuredBreach.DataClasses }}</span>
```

Comment:
Did we mean to remove the space after the “:” in there, or is the space being added via CSS?

Reply:
Added via CSS and I just took out the same space in breach_listing as well.